scripts/rollup/build-all-release-channels.js

```diff
@@ -22,6 +22,20 @@ const sha = (
   spawnSync('git', ['show', '-s', '--format=%h']).stdout + ''
 ).trim();
 
+const {stdout} = execSync(`git show -s --format=%ci ${sha}`);
```

Comment:
Could also pass a format `%Y%m%d`, instead of doing that in JavaScript below

Reply:
Yeah, good call. I _think_ the way to do that would be:
```sh
git show -s --format=%cd --date=format:'%Y%m%d' <sha>
```
I'll change that.